Packs/HarfangLabEDR/Integrations/Hurukai/Hurukai.yml

```diff
@@ -137,7 +137,7 @@ script:
       description: HarfangLab job action
   - arguments:
     - name: agent_id
-      description: Agent unique identifier as provided by the HarfangLab EDR Manager
```

Comment:
@AradCarmi - can you remove this change to another pack?

Reply:
Sure it was a mistake, nice catch!